src/components/App/components/UserInput/UserInput.js

```diff
@@ -71,7 +71,6 @@ function UserInput(props) {
                   id: 'font-size',
                   type: 'number',
                   min: MIN_FONT_SIZE_IN_PX,
-                  max: MAX_FONT_SIZE_IN_PX,
```

Comment:
Oh no! I didn't mean to remove the max here. I thought we are removing it only from the Preview?